values.yaml

```diff
@@ -992,6 +992,9 @@ connectInject:
         memory: "150Mi"
         cpu: "50m"
 
+controller:
+  enabled: false #todo
```

Comment:
There is a todo here and also missing docs. What about the other helm values from the RFC?

Reply:
The plan is to add them later. I've taken note of this task.

Reply:
Ah ok, cool. Thanks for clarifying.